### ISSUE TITLE
Support portable version of torrent clients

### ISSUE THREAD

**Issue by taazz** [DavidMoore/ipfilter#28]
It would be helpfull if you could add support for portable version of the various torrent clients you already support and since that will be a loosing battle on your part add a way for the end user to define a list paths that the filter should be copied after download.

**Comment by raaomoka** [comment]
Yeah, the earlier version (with command line UI) was working fine with my portable uTorrent, then that stopped working, I found the latest here on GitHub, and sadly, the latest doesn't seem to work as easily. Have to manually copy the ipfilter.dat file from %AppData%\DavidMoore\IPFilter to my portable uTorrent directory...or sync it to my portable uTorrent directory using FreeFileSync...either way, still adding another step to the process.
